lib/reply.js

```diff
@@ -237,6 +237,21 @@ Reply.prototype.getResponseTime = function () {
   return responseTime
 }
 
+Reply.prototype.then = function (fullfilled, rejected) {
```

Comment:
Please add a comment to this method to describe its use I can imagine that's hard to grab for new collaborators.

Reply:
I agree, a comment here, maybe with a link to the original issue would help.